app/views/games/show.html.erb

```diff
@@ -3,98 +3,20 @@
 
 
 <div class="board-container">
-    <table id="board">
-    <table style="width: 100%">
+    <table id="board" style="width: 100%, height: auto">
     <% 8.times.each do |row| %>
-      <tr>
+      <tr id="row">
         <% 8.times.each do |col| %>
           <% piece = @game.pieces.where(position_x: col, position_y: row).first %>
           <% if piece.present? %>
-            <td class="#pawn_spot" piece_id="<%= piece.id %>" piece_name="pawn">
+            <td id="<%= piece.color %><%= piece.type %>" data-piece_id="<%= piece %>" data-piece_name="<%= piece.type %>">
               <%= piece.position_x %>, <%= piece.position_y %>
             </td>
           <% else %>
             <td></td>
           <% end %>
         <% end %>
-        <td></td>
-        <td></td>
-        <td></td>
-        <td></td>
-        <td></td>
-        <td></td>
       </tr>
```

Comment:
Lets be sure to merge this in after my pull request, or mine will undo some of this piece data we're putting into the table